### DISCUSSION TITLE
How to prevent General/About window from always appearing on login

### DISCUSSION THREAD

**Discussion by acidburst2** [dwarvesf/hidden#188] [Q&A]
Hello,

I would like to know if there is a setting I am missing that blocks the General/About window from always launching when I log into computer? I like that the app will launch with each login, but I would really like this window to appear minimized or just not appear at all. Is this possible? See attached screenshot of window I am referring to.

![Screen Shot 2022-01-08 17 56 10 PM](https://user-images.githubusercontent.com/97371011/148666248-95c37d37-8cc3-4823-91f2-3b79cd67e172.png)

Thanks

**Comment by darktweek** [comment]
I have the same issue.
Have you manage to find a solution ?
